config/crd/bases/topology.nephio.org_nftopologies.yaml

```diff
@@ -165,6 +186,48 @@ spec:
             type: object
           status:
             description: NFTopologyStatus defines the observed state of NFTopology
+            properties:
+              conditions:
+                description: Current service state of the NF.
+                items:
+                  type: string
+                type: array
+              nfInstances:
+                description: Detail on the deployed instances.
+                items:
+                  description: NFDeployedInstance defines an NF instance that is deployed
+                  properties:
+                    clusterName:
+                      description: name of workload cluster where the NF instance
+                        is to be deployed
+                      type: string
+                    connectivities:
+                      description: "TODO(s3wong): the following parameters should
```

Comment:
Is there a better way to track this rather then a TODO?

Reply:
I removed this TODO on the update. I commented it out in this patch anyway, so just removing it would be fine.